WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/MySitesPage.kt

```diff
@@ -283,21 +283,22 @@ class MySitesPage {
     }
 
     private fun scrollToCard(elementID: Int): MySitesPage {
-        WPSupportUtils.waitForElementToBeDisplayed(elementID)
```

Comment:
I added this with #18345, and I have no idea why, it makes no sense 😕. This worked only because the Domains card was already on-screen, and did not require scrolling.